WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/EUCustomsScenarioValidator.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+import Yosemite
+
+/// Validation logic for Shipping scenarios with specific EU Customs.
+///
+/// Refer to [USPS instructions](https://www.usps.com/international/new-eu-customs-rules.htm) for more context.
+///
+class EUCustomsScenarioValidator {
```

Comment:
Tip: If you are not planning to store any state this could be a `Struct`

Reply:
Oh, thanks for the tip! I keep creating confusion about when to use class or structs when using Swift 🤦. I adjusted this in the [following PR](https://github.com/woocommerce/woocommerce-ios/pull/9704) since I was changing it a lot there.